src/style/color.rs

```diff
@@ -390,6 +390,41 @@ impl Color {
         // Delegate to the function for normalized HSL to RGB conversion
         normalized_hsl_to_rgb(h / 360.0, s / 100.0, l / 100.0)
     }
+
+    /// Converts a `HSLuv` representation to a `Color::Rgb` instance.
+    ///
+    /// The `from_hsluv` function converts the Hue, Saturation and Lightness values to a
+    /// corresponding `Color` RGB equivalent.
+    ///
+    /// Hue values should be in the range [0, 360].
+    /// Saturation and L values should be in the range [0, 100].
+    /// Values that are not in the range are clamped to be within the range.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use ratatui::prelude::*;
+    ///
+    /// let color = Color::from_hsluv(360.0, 50.0, 75.0);
+    /// assert_eq!(color, Color::Rgb(223, 171, 181));
+    ///
+    /// let color: Color = Color::from_hsluv(0.0, 0.0, 0.0);
+    /// assert_eq!(color, Color::Rgb(0, 0, 0));
+    /// ```
+    #[cfg(feature = "palette")]
+    pub fn from_hsluv(h: f64, s: f64, l: f64) -> Self {
+        use palette::{Clamp, FromColor, Hsluv, Srgb};
+
+        let hsluv = Hsluv::new(h, s, l).clamp();
```

Comment:
Hsluv supports more than just f64. It defaults to f32, and supports other numeric types like u8 for instance. It also supports a whitepoint. I wonder whether we should consider providing methods to convert from palette types instead of handling specific color spaces.

Reply:
I'd like to get this in for the next release - would it be fine if we discuss in a separate issue?